### PR TITLE
Docs for iZone component.

### PR DESCRIPTION
**Pull request by Swamp-Ig**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#24550

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/pizone.markdown [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- Via the integrations configuration of the Home Assistant user interface.
+- Or via the `configuration.yaml` file by adding the following:
+
+If you load the integration from the integrations window, all local iZone instances are added. The integration will discover any new instances added to the local area later on as well.
+
+Alternately if there is more than one iZone system on the local network and one or more must be excluded use manual configuration:
+
+```yaml
+# Full manual example configuration.yaml entry
+izone:
```
Comment (klaasnicolaas) [inline source/_components/pizone.markdown:29]: I mis configuration variables explanation 😉 https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/pizone.markdown [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+The `iZone` component allows access of control of local [iZone](https://izone.com.au/) ducted reverse-cycle climate control devices. These are largely available in Australia.
+
+## {% linkable_title Supported hardware %}
+
+Any current iZone unit with ducted reverse cycle airconditioning and the CB wired or wireless bridge device installed should currently work. There is currently no support for the iZone lights, reticulation, or other devices.
+
+## {% linkable_title Configuration %}
+
+The Daikin component can be configured in two ways.
```
Comment (zkrebbekx) [inline source/_components/pizone.markdown]: Should this say iZone instead of Daikin?

**Comment by Swamp-Ig** [comment]
Changes made as requested

File: source/_components/pizone.markdown [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+layout: page
+title: "iZone climate control"
+description: "Instructions on how to integrate iZone climate control devices with Home Assistant."
+date: 2019-06-15 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Climate
+ha_release: 0.95
```
Comment (klaasnicolaas) [inline source/_components/pizone.markdown]:
```suggestion
ha_release: 0.96
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/pizone.markdown [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+layout: page
+title: "iZone climate control"
+description: "Instructions on how to integrate iZone climate control devices with Home Assistant."
+date: 2019-06-15 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Climate
+ha_release: 0.96
```
Comment (MartinHjelmare) [inline source/_components/pizone.markdown]: "0.100"

**Review by MartinHjelmare — DISMISSED** [review]

**Review by klaasnicolaas — DISMISSED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @Swamp-Ig! 👍

**Pull request merged**